src/main/java/com/example/cecv_e_commerce/controller/CategoryController.java

```diff
@@ -0,0 +1,25 @@
+package com.example.cecv_e_commerce.controller;
+
+import com.example.cecv_e_commerce.service.CategoryService;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.example.cecv_e_commerce.domain.dto.category.CategoryListResponseDTO;
+
+@RestController
+@RequestMapping("/categories")
+public class CategoryController {
+
+    private final CategoryService categoryService;
+
+    public CategoryController(CategoryService categoryService) {
+        this.categoryService = categoryService;
+    }
```

Comment:
Chị thấy em có dùng lombok, vậy sao mình ko kiệm code thêm tí nữa nhỉ
```suggestion
@RequiredArgsConstructor
public class CategoryController {

    private final CategoryService categoryService;

```